### DISCUSSION TITLE
Layout for mobile screens

### DISCUSSION THREAD

**Discussion by Temder** [painebenjamin/app.enfugue.ai#127] [Ideas]
Hi!

I want to ask if a layout for smaller screens is planned. Or maybe I didn't find it yet.
For now I am experimenting with the site layout and adding media queries to the css scripts.

This is what it looks like for now:

<img src="https://github.com/painebenjamin/app.enfugue.ai/assets/57451447/14d5d220-90fb-4e63-8005-61f9e3845924" width="200px"/>

**Comment by painebenjamin** [comment]
Hello @Temder, thank you for the suggestion!

I've always had a mobile interface in sight, but I still need to prioritize it. I'm preparing the builds of `0.3.3` with some minor bug fixes and DragNUWA, and then the plan is to begin work on `0.4.0`, which will have a significant front-end layout change - I'll make sure that small screens are accounted for.

**Comment by Temder** [comment]
Thanks for the response. Until then I will use my method
